File: src/main/java/com/bot/utils/DbHelpers.java
package com.bot.utils;

import java.sql.Connection;
import java.sql.PreparedStatement;
import java.sql.ResultSet;
import java.sql.SQLException;

public class DbHelpers {

    public static void close(PreparedStatement preparedStatement, ResultSet resultSet, Connection connection) {
        try {
            if (preparedStatement != null)
                preparedStatement.close();
            if (resultSet != null)
                resultSet.close();
            if (connection != null)
                connection.close();

        } catch (SQLException e) {
            e.printStackTrace();
        }
    }
}

File: src/main/java/com/bot/utils/DbHelpers.kt
package com.bot.utils

import java.sql.Connection
import java.sql.PreparedStatement
import java.sql.ResultSet
import java.sql.SQLException

object DbHelpers {

    fun close(preparedStatement: PreparedStatement?, resultSet: ResultSet?, connection: Connection?) {
        try {
            connection?.close()
            preparedStatement?.close()
            resultSet?.close()
        } catch (e: SQLException) {
            e.printStackTrace()
        }

    }
}
